src/aatoolbox/datasources/ecmwf/combine_seas5_monthly.py

```diff
@@ -0,0 +1,103 @@
+"""
```

Comment:
The reason why it is important and needs to be saved is because in `pa-anticipatory-action` we use it again. But there we also have the `CDS` source so the user can choose whether to use this data or the cds data. If choosing to use directly the ecmwf data it makes sense imo that you get one dataset that contains the combined api and realtime data. Would you agree? 

It doesn't take super long to run so maybe somehow we don't have to save it if we port everything to `aa-toolbox` but not sure if that will work

Would probably run it for each country that uses the ecmwf data

Reply:
> get one dataset that contains the combined api and realtime data

I guess my hesitation is that it doesn't really fit into the paradigm of what `datasource` is supposed to be, i.e. the download, cleaning, and loading of data from a single source. The combination of sources feels more like post-processing to me. Do you think the results could be saved in exploration?

Uff these edge cases are always the hardest :sweat_smile: I will think about it a bit more.